[2022-01-25 06:36:02,647] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-23T00:00:00+00:00 [queued]>
[2022-01-25 06:36:02,658] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-23T00:00:00+00:00 [queued]>
[2022-01-25 06:36:02,658] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:36:02,658] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 06:36:02,659] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:36:02,671] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-23 00:00:00+00:00
[2022-01-25 06:36:02,676] {standard_task_runner.py:52} INFO - Started process 3905 to run task
[2022-01-25 06:36:02,680] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-23T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp8r8xn18r', '--error-file', '/tmp/tmp3b2dvr_k']
[2022-01-25 06:36:02,684] {standard_task_runner.py:77} INFO - Job 13: Subtask first_task
[2022-01-25 06:36:02,748] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-23T00:00:00+00:00 [running]> on host ceab856ace11
[2022-01-25 06:36:02,912] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-23T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-23T00:00:00+00:00
[2022-01-25 06:36:02,913] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 06:36:02,913] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 06:36:02,922] {subprocess.py:85} INFO - Output:
[2022-01-25 06:36:02,925] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 06:36:02,925] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 06:36:02,951] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220123T000000, start_date=20220125T063602, end_date=20220125T063602
[2022-01-25 06:36:02,972] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 06:36:03,001] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-25 07:27:45,787] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-23T00:00:00+00:00 [queued]>
[2022-01-25 07:27:45,799] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-23T00:00:00+00:00 [queued]>
[2022-01-25 07:27:45,799] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:45,799] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 07:27:45,799] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:45,812] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-23 00:00:00+00:00
[2022-01-25 07:27:45,816] {standard_task_runner.py:52} INFO - Started process 366 to run task
[2022-01-25 07:27:45,820] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-23T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpwbjw_26f', '--error-file', '/tmp/tmpqplyxpdb']
[2022-01-25 07:27:45,824] {standard_task_runner.py:77} INFO - Job 13: Subtask first_task
[2022-01-25 07:27:45,994] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-23T00:00:00+00:00 [running]> on host 943a4c094194
[2022-01-25 07:27:46,046] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-23T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-23T00:00:00+00:00
[2022-01-25 07:27:46,047] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 07:27:46,047] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 07:27:46,055] {subprocess.py:85} INFO - Output:
[2022-01-25 07:27:46,058] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 07:27:46,058] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 07:27:46,082] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220123T000000, start_date=20220125T072745, end_date=20220125T072746
[2022-01-25 07:27:46,114] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 07:27:46,142] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
